### PR TITLE
Send the account creation email back to the user after 7 days

### PR DESCRIPTION
**Pull request by gregory-latinier**
This is to fix the following issue: https://github.com/steemit/faucet-admin/issues/30

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -49,6 +50,31 @@ setInterval(() => {
   });
 }, 60 * 60 * 1000);
 
+async function sendReminderEmail() {
+  const users = await db.users.findAll({
+    where: { updated_at: { [Op.lt]: moment().subtract(7, 'days').toDate() } },
```
Comment (jnordberg) [inline app.js]: This would send out emails to unconfirmed users as well as users who already created their account

File: app.js [hunk 2]
```diff
@@ -49,6 +50,31 @@ setInterval(() => {
...
+  });
+
+  users.map(async (user) => {
+    const mailToken = jwt.sign({
+      type: 'create_account',
+      email: user.email,
+    }, process.env.JWT_SECRET, { expiresIn: '7d' });
+    await mail.send(user.email, 'create_account', {
```
Comment (jnordberg) [inline app.js]: We should have a specific template for the reminder email

File: app.js [hunk 3]
```diff
@@ -49,6 +50,31 @@ setInterval(() => {
...
+      updated_at: new Date(),
+    }, { where: { email: user.email } });
+  });
+}
+setInterval(() => {
+  logger.debug('running email reminder');
+  sendReminderEmail().catch((error) => {
+    logger.error(error, 'error sending email reminder');
+  });
+}, 60 * 60 * 1000);
```
Comment (jnordberg) [inline app.js]: Offset this from the cleanup db timer, bad luck having them both run at the same time

**Comment by gregory-latinier** [comment]
Could you be more precise ? What curl request ?

**Comment by gl2748** [comment]
I'll tidy this up tomorrow. The new email expiry is 14 days.
Relates to #259

**Comment by gl2748** [comment]
Closing this. We will manually send this email, see #283

**Pull request closed without merging**